torch/csrc/jit/runtime/vararg_functions.cpp

```diff
@@ -39,6 +39,108 @@ void format(Stack& stack, size_t num_inputs) {
   push(stack, ss.str());
 }
 
+// IValue tags are intentionally private, so we need additional logic to cast
+// the IValue type to the specified format.
+std::string getFormattedArg(char key, const IValue& ival, int precision=6) {
+  //TODO: Implement precison-based formatting
+  std::stringstream ss;
+  bool added = false;
```

Comment:
Using `added` is a bit confusing, I think having `ss << ival` in every case is perfectly fine.